<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnplGeneralForm</class>
 <widget class="QDialog" name="QnplGeneralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>User interface</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Language:    </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Português (Brasil)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Español</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>* Application restart required</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Screen Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>640x480 (4:3)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800x600 (4:3)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024x768 (4:3)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>854x480 (16:9)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280x720 (16:9)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920x1080 (16:9)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/tv</normaloff>:/icon/tv</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>320x400 (4:5)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/mobile</normaloff>:/icon/mobile</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400x320 (5:4)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/mobile</normaloff>:/icon/mobile</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>320x180 (16:9)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/mobile</normaloff>:/icon/mobile</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>320x240 (4:3)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/mobile</normaloff>:/icon/mobile</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>240x320 (3:4)</string>
         </property>
         <property name="icon">
          <iconset resource="../application.qrc">
           <normaloff>:/icon/mobile</normaloff>:/icon/mobile</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>System</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="toolTip">
         <string>Automatically start the presentation when a document is opened</string>
        </property>
        <property name="text">
         <string>Autoplay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
